package.json

```diff
@@ -34,10 +34,10 @@
     "tap": "^16.0.1"
   },
   "engines": {
-    "node": ">=12.0"
+    "node": ">=14.0"
   },
   "peerDependencies": {
-    "newrelic": ">=6.11.0"
+    "newrelic": ">=8.7.0"
```

Comment:
FYI: this is fine, but we will have to do another PR after we ship 9.x of the agent to update these

Reply:
When do we have 9.x scheduled?

Reply:
we typically don't schedule it but it's the Node 18 MMF. maybe later this week, or early next?

Reply:
I'm going to revert this in the release prep